### ISSUE TITLE
Add tests for `utils.py`

### ISSUE THREAD

**Issue by Gonmeso** [gizatechxyz/actions-sdk#35]
Currently, only `models.py` and `agents.py` have some kind of testing, the focus of this issue is to add tests to the `utils.py` script.

Some tips on this:

* Use `patch` or `patch.object` for the clients
* When patching return a value of the schema needed. These schemas can be found in the [cli repo](https://github.com/gizatechxyz/giza-cli/tree/main/giza/schemas)
* Make sure to test exceptions

An example:

```python
from giza.schemas import SchemaClass

schema_object = SchemaClass(
    ...
)

@patch("giza_actions.utils.ClientToPatch.method", return_value=schema_object)
def test_something(mock_schema):
    # Test Here
    ...
    # Assert result
    mock_schema.assert_called_once()
```

**Comment by stevencartavia** [comment]
Hi, can I take this?

**Comment by adrianvrj** [comment]
Hello, can I hop on this?

**Comment by EmmanuelAR** [comment]
Hey @Gonmeso , i can help too to add some test!!
Is still posible?
We can, between the 3 of us (we are from Costa Rica), divide the methods to test them, it could be a solution

**Comment by raphaelDkhn** [comment]
Hey guys ODHack just opened, I can assign it to you :)

**Comment by EmmanuelAR** [comment]
Sure, excellent, let us create a group and divided the work, thanks @raphaelDkhn!!

**Comment by EmmanuelAR** [comment]
Heres my Pr from my part team https://github.com/gizatechxyz/actions-sdk/pull/43

**Comment by Gonmeso** [comment]
Fixed in #45 #43 #42
